config/core/configmaps/deployment.yaml

```
@@ -76,6 +76,15 @@ data:
     # If omitted, no value is specified and the system default is used.
     queue-sidecar-ephemeral-storage-limit: "1024Mi"
 
+    # Sets tokens associated with specific audiences for the queue proxy's QPOptions
+    # If omitted, or empty, no tokens are created
+    # Supports a list of audiences
+    queue-sidecar-tokens: ""
+
+    # Sets rooCA for the queue proxy's QPOptions
+    # If omitted, or empty, no rooCA is added to the golang rooCAs
+    queue-sidecar-rootca: ""
```

```suggestion
    # Sets rootCA for the queue proxy's QPOptions
    # If omitted, or empty, no rootCA is added to the golang rootCAs
    queue-sidecar-rootca: ""
```

Also, are we expecting a path or a name here?